tools/train.py

```diff
@@ -62,6 +62,33 @@ def parse_args():
     return args
 
 
+def determine_max_batch_size(model, cfg, min_bs=2, max_bs=512, step=2):
+    cuda_model = model.cuda()
+
+    img_shape = list([t for t in cfg.data.train.dataset.pipeline if t['type'] == 'Resize'][0]['img_scale'])
```

Comment:
Actual image blob size depend on the `Pad` stage of the preprocessing pipeline. During export we run all preprocessing stages on a (dummy) input image. Here you just use the shape as is, which might be wrong.

Reply:
fixed